packages/slice-machine/components/AppLayout/Navigation/ChangesListItem.tsx

```diff
@@ -36,7 +40,7 @@ export const ChangesListItem = forwardRef<HTMLLIElement, ChangesListItemProps>(
     return (
       <HoverCard
         open={open}
-        openDelay={1500}
+        openDelay={3000}
```

Comment:
This is intentional right?

Reply:
Yes: https://prismic-team.slack.com/archives/C02L3FN3AJK/p1690534832385109?thread_ts=1690529552.348829&cid=C02L3FN3AJK